src/elektra/redshift.ini

```diff
@@ -0,0 +1,308 @@
+opt/arg = required
+
+[fade]
+type = boolean
+description = Whether transitions between color temperatures (e.g. from daytime to nighttime) should happen fast or slow.
+default = 1
+example = 0
+opt = f
+opt/long = fade
+opt/arg = none
+
+[brightness/day]
+type = float
+description = The screen brightness during daytime. If both day and night brightness are set, these will overrule the value of brightness.
+default = 1.0
+example = 0.8
+check/type = float
+check/range = 0-1
+opt/long = brightness-day
+opt/arg = required
+
+[brightness/night]
+type = float
+description = The screen brightness during nighttime. If both day and night brightness are set, these will overrule the value of brightness.
+default = 1.0
+example = 0.8
+check/type = float
+check/range = 0-1
+opt/long = brightness-night
+opt/arg = required
+
+[gamma/day]
+type = string
+description = The screen gamma during daytime. Supported formats: 1. One value, that will be used for red, green and blue (e.g. 0.9). 2. Three colon-separated values for red, green and blue respectively (e.g. 0.9:0.9:0.9).
+default = 1.0:1.0:1.0
+example = 0.9
+; Regex ensures format "float" or "float:float:float". Adapted from https://www.regular-expressions.info/floatingpoint.html
+check/validation = ^([0-9]*[\.,]?[0-9]+)(:([0-9]*[\.,]?[0-9]+):([0-9]*[\.,]?[0-9]+))?$
+opt/long = gamma-day
+opt/arg = required
+
+[gamma/night]
+type = string
+description = The screen gamma during nighttime. Supported formats: 1. One value, that will be used for red, green and blue (e.g. 0.9). 2. Three colon-separated values for red, green and blue respectively (e.g. 0.9:0.9:0.9).
+default = 1.0:1.0:1.0
+example = 0.9
+; Regex ensures format "float" or "float:float:float". Adapted from https://www.regular-expressions.info/floatingpoint.html
+check/validation = ^([0-9]*[\.,]?[0-9]+)(:([0-9]*[\.,]?[0-9]+):([0-9]*[\.,]?[0-9]+))?$
+opt/long = gamma-night
+opt/arg = required
+
+[gamma/preserve]
+type = boolean
+description = Use to preserve existing gamma ramps before applying adjustments.
+default = 1
+example = 0
+opt = P
+opt/arg = none
+
+[adjustment/method]
+type = enum
+check/enum = #7
+check/enum/#0 = drm
+check/enum/#1 = dummy
+check/enum/#2 = quartz
+check/enum/#3 = randr
+check/enum/#4 = vidmode
+check/enum/#5 = w32gdi
+check/enum/#6 = auto
+check/enum/#7 = list
+description = The method used to adjust screen color temperature. By default, one of the supported methods on the current OS will be chosen automatically. For details see section "Alternative Features" in file DESIGN in root directory.
+default = auto
+example = randr
+; default = w32gdi - For Windows the default is the only supported method 'w32gdi'
+; default = randr - For Linux the default is 'randr'
+; default = quartz - For macOS the default is 'quartz'
+opt = m
+opt/long = method
+opt/arg = required
+
+[adjustment/method/help]
+type = boolean
+description = Prints the help of the adjustment methods.
+default = 0
+example = 1
+opt/long = help-methods
```

Comment:
Not working? `./src/redshift --help-methods`

Reply:
Please see https://github.com/jonls/redshift/pull/837#discussion_r706612433 , it might be the same cause+fix.